examples/agentInteractionsDemo/app.js

```diff
@@ -158,19 +158,6 @@ new Vue({
             return;
         }
 
-        /*
-            * Note: To use this app in your own org, you will need to create your own OAuth2 Client(s)
-            * in your PureCloud org.  After creating the Implicit grant client, map the client id(s) to
-            * the specified region key(s) in the object below, deploy the page, and configure an app to point to that URL.
-            */
-        let pcOAuthClientIds = {'mypurecloud.com': 'implicit-oauth-client-id-here'};
-
-        let clientId = pcOAuthClientIds[pcEnvironment];
-        if (!clientId) {
-            this.errorMessage = pcEnvironment + ': Unknown/Unsupported PureCloud Environment';
```

Comment:
Looks like all of these examples were already including the dynamic pcEnvironment value in their error messages.  The frameworks take care of it, but I'm always cognizant of XSS when we're using raw dom apis.  I believe I caught all the innerHTMLs, etc. when the examples went through initial PR, but we should give this a once over just to confirm.

Reply:
Confirmed that all of these examples just present it as text, not html.  So, g2g